src/index.js

```diff
@@ -141,3 +140,5 @@ export { default as version } from './version';
 export {
   default as ContentNotification,
 } from './components/notifications/content-notification';
+
+export * from './hooks';
```

Comment:
Extra unrelated little change.

Reply:
Does UIKit really want to export it's hooks already? This would maybe even a feature releasea as a result.

Reply:
It was already exposed actually, I just refactored it to `export *` instead of `export { default as useToggleState }`

Reply:
Hm. I, personal opinion ahead, liked it before more then. The library opts into what it wants to expose instead of defaulting to everything.

Reply:
Sure my bad, I'll revert this change then

Reply:
https://github.com/commercetools/ui-kit/pull/965/commits/8361bc17e7c9cd7d84db70adf633bceb0d1c5821